<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IlacListe</class>
 <widget class="QDialog" name="IlacListe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>İlaçlar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblAra">
       <property name="text">
        <string>Ara</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAra"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnYeni">
       <property name="text">
        <string>Yeni İlaç</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnBilgi">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Bilgi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSil">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sil</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
